### DISCUSSION TITLE
mailmap (git commands and relocation)

### DISCUSSION THREAD

**Discussion by spaette** [community/community#34833] [Other Feature Feedback, Questions, & Ideas]
Relocating a repository's top level `.mailmap` file to `.github/.mailmap`.

Top level GitHub repository files appear to be indexed by search engines.

What might comprise a list of `git` commands + options to test if relocation wouldn't result in any loss of functionality?

**Comment by airtower-luna** [comment]
For **Git** (without "Hub") `.github/` is just like any other directory in your repository. So if you move the `.mailmap` file it'll be ignored. You *can* configure the location of the mailmap file to use, but then every user would have to do that in their local repository.

See: https://git-scm.com/docs/gitmailmap

**Comment by spaette** [comment]
@apelisse

Can you advise on this?

> If we move to `.github`, will `git` know what to do?

This is a question posed by an Organization Member.

The file .git/index contains the location of the .mailmap file if the latter exists.

**Reply by apelisse** [reply]
I think @airtower-luna has the right answer.
